docs/spec/auth/token.md

```diff
@@ -148,6 +148,9 @@ challenge, the client will need to make a `GET` request to the URL
         there is more than one <code>scope</code> entry from the <code>WWW-Authenticate</code>
         header. The above example would be specified as:
         <code>scope=repository:samalba/my-app:push</code>.
+	* `PUT` or `docker push` request need authenticate actions of `[pull,push]`;
```

Comment:
This doesn't really belong here. If there is a section on the interpretation of the scope on the registry, that is where this should belong.